ibm/mas_devops/roles/suite_install/tasks/main.yml

```diff
@@ -1,12 +1,90 @@
 ---
 
-# 1. Check for undefined properties that do not have a default
+# 1.a Check for undefined properties that do not have a default
 # -----------------------------------------------------------------------------
 - name: "Fail if mas_instance_id is not provided"
   assert:
     that: mas_instance_id is defined and mas_instance_id != ""
     fail_msg: "mas_instance_id property is required"
 
+# 1.b Set facts for mas_annotations_dict, HyperscalerFormat, HyperscalerChannel
+# -----------------------------------------------------------------------------
+- name: "Set facts : mas_annotations_dict"
+  set_fact:
+    mas_annotations_dict: "{{ mas_annotations | string  | ibm.mas_devops.getAnnotations() }}"
+
+- name: "Set facts : HyperscalerFormat"
+  set_fact:
+    input_hyperscalerFormat: "{{ item.value }}"
+  when:
+    -  mas_annotations_dict is defined and item.key == 'mas.ibm.com/hyperscalerFormat'
+  with_dict:
+    - "{{  mas_annotations_dict  }}"
+
+- name: "Set facts : HyperscalerChannel"
+  set_fact:
+    input_hyperscalerChannel: "{{ item.value }}"
+  when:
+    -  mas_annotations_dict is defined and item.key == 'mas.ibm.com/hyperscalerChannel'
+  with_dict:
+    - "{{  mas_annotations_dict  }}"
+
+- name: "Set facts : HyperscalerProvider"
+  set_fact:
+    input_hyperscalerProvider: "{{ item.value }}"
+  when:
+    -  mas_annotations_dict is defined and item.key == 'mas.ibm.com/hyperscalerProvider'
+  with_dict:
+    - "{{  mas_annotations_dict  }}"
+
+- name: "Debug input hyperscaler values"
+  debug:
+    msg:
+      - "Hyperscaler Provider ............ {{ input_hyperscalerProvider | default('<undefined>', true) }}"
```

Comment:
See style guidelines re: debug messages https://github.com/ibm-mas/ansible-devops/blob/master/CONTRIBUTING.md#align-debug-messages

Reply:
Hi David, Incorporated the style guidelines for debug messages.